pkgs/applications/misc/octoprint/default.nix

```diff
@@ -14,6 +14,20 @@ let
     self = py;
     packageOverrides = lib.foldr lib.composeExtensions (self: super: { }) (
       [
+        (
+          self: super: {
+            flask-limiter = super.flask-limiter.overridePythonAttrs (oldAttrs: rec {
+              version = "2.6.2";
+              src = fetchFromGitHub {
+                owner = "alisaifee";
+                repo = "flask-limiter";
+                rev = version;
+                sha256 = "sha256-eWOdJ7m3cY08ASN/X+7ILJK99iLJJwCY8294fwJiDew=";
+              };
+              doCheck = false;
```

Comment:
Please add a comment about why the checks are disabled.

Reply:
The shouldn't be disabled. I must have overlooked that line.